lib/exercism.rb

```diff
@@ -14,7 +14,8 @@
 require 'exercism/daily'
 require 'exercism/daily_count'
 require 'exercism/deleted_iterations'
-require 'exercism/github'
+require 'exercism/github_user'
+require 'exercism/github_auth'
```

Comment:
I'd like to see only `github_auth` being required here.
`github_user` is something that we shouldn't need to care about outside the context of authentication.